lib/line/bot/client.rb

```diff
@@ -590,6 +590,127 @@ def get_friend_demographics
         get(endpoint, endpoint_path, credentials)
       end
 
+      # Create an audience group by uploading user_ids
+      #
+      # Parameters are described here.
+      # https://developers.line.biz/en/reference/messaging-api/#create-upload-audience-group
+      #
+      # @param opts [Hash] options
+      #
+      # @return [Net::HTTPResponse] This response includes an audience_group_id.
+      def create_user_id_audience(params)
+        channel_token_required
+
+        endpoint_path = '/bot/audienceGroup/upload'
+        post(endpoint, endpoint_path, params.to_json, credentials)
```

Comment:
Is it a PUT correctly?
https://developers.line.biz/en/reference/messaging-api/#http-request-20